Topics/Software_Engineering/Scrum Planning Meeting.md

```diff
@@ -0,0 +1,32 @@
+
+
+## Integrating Sprint Planning Meetings in CSC301 
+*Consider an example of how a Scrum Planning Meeting can be integrated into our course’s workflow.*
+
+### Before the Planning Meeting:
+After a Product Manager and Scrum Master are chosen in Deliverable 1, consider each deliverable (deliverable 2 to 5) equivalent to a sprint, with each sprint having one dedicated Sprint Planning Meeting. As a ScrumMaster and Product Manager, organize an hour long meeting within the next week after the Deliverable handout is released. Prior to the Scrum Planning Meeting, you both should also read through the handout and identify features/Product Backlog items for the upcoming sprint and add them to the Product Backlog list on Jira (or the team’s chosen product management tool).
+
+### During the Planning Meeting: 
+Start by reminding the team of the upcoming Deliverable and review the definition of “Done”, which in this case would be meeting both the rubric and your partner's requirements. As a team, take this opportunity to brainstorm more tasks that need to be completed based on what your partner has asked and add them into the Product Backlog list.
+
+Discuss how much time would be needed for each Product Backlog item listed on Jira and based on this estimate, determine items for the Sprint Backlog. Following, select items from the Product Backlog and add them into the Sprint Backlog, allow team members to assign themselves to sprint tasks (including yourself) and discuss any further action items as a team. 
+
+Once the Sprint Backlog items are finalized, as the ScrumMaster, call for a group consensus on the plan and conclude the meeting.
+
+## Key Takeaway
+The Sprint Planning Meeting lays the foundation for the sprint, helps the team understand their tasks and sets the stage for effective Scrum meetings in the weeks ahead. This meeting ensures everyone is aligned and ready for the work to come, making future collaborations smoother and more productive.
+
+## Resources
+- [What is Sprint Planning](https://www.atlassian.com/agile/project-management/project-management-intro)
+- [Sprint Planning Learning Series](https://www.scrum.org/learning-series/sprint-planning)
+- [Sprint Planning Meeting: A Simple Cheat Sheet](https://www.wrike.com/blog/set-up-your-waterfall-workflow/)
```

Comment:
This link directs you to a page for "How to Set Up Your Waterfall Workflow", which may not be very relevant to your topic.
Is the link supposed to be the following instead? 
https://www.wrike.com/scrum-guide/scrum-sprint-planning/